sources/packages/backend/workflow/src/workflow-definitions/fulltime-assessment-2025-2026.bpmn

```
@@ -2165,8 +2165,8 @@ There is also a per application limit based on the number of weeks in the offeri
         <zeebe:calledDecision decisionId="dmnFullTimeMSOLAllowance" resultVariable="dmnFullTimeStudentMSOLAllowance" />
         <zeebe:ioMapping>
           <zeebe:input source="=&#34;student&#34;" target="lookupType" />
+          <zeebe:output source="=if offeringDelivered = &#34;online&#34;&#10;  or offeringDelivered = &#34;blended&#34;&#10;  or (studentDataLivingWithPartner = &#34;no&#34; &#10;  and studentDataRelationshipStatus = &#34;married&#34;) &#10;then&#10;  &#34;BC&#34;&#10;else&#10;  institutionLocationProvince" target="institutionLocationProvince" />
```

If the below follows a policy direction similar to which is meaningful from a business perspective, it would be ok, otherwise the "BC" fallback seems misleading.

> If distance education/blended learning program or if married and claiming separate residence use B.C. to calculate MSOL even if school is out of province.

---

That language is straight from our policy team about the expectation on which location Province to use for MSOL. Are you suggesting I use a "calculated" province based on those conditions to preserve institutionLocationProvince?